lib/libvfio-user.c

```diff
@@ -576,10 +576,6 @@ handle_device_get_region_io_fds(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg)
 
     vfu_reg = &vfu_ctx->reg_info[req->index];
 
-    if (vfu_reg->fd == -1) {
```

Comment:
I'm lacking some context, IIUC the problem is that we don't allow a region to have ioregionfd sub-regions if the region is not memory mappable? ioregionfd and memory-mappable region are orthogonal, right?

Reply:
right, exactly.

Reply:
Could we document that on the commit message?

Reply:
I tried to clarify the commit message a bit more?

Reply:
You did, I only looked at the PR title.. my bad.